.gitattributes

```diff
@@ -0,0 +1 @@
+*.theme.json -diff
```

Comment:
We don't do this elsewhere in our whiskers generated ports, I'm not really sure I see the reasoning for this here? 
Does this impact the GitHub PR diff view?

Reply:
Oh I see. I normally do this for files that are generated so that they're not
diffed locally. Since I will regenerate them from source, if there are
conflicts. I think it does sometimes impact the GitHub UI diff view. According
to [their docs][gh-docs], GH won't render a file if you include the
`linguist-generated=true` attribute which might be more appropriate here?.

[gh-docs]: https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github

I can remove it if it doesn't make sense. I added it to discuss it but it's
mostly muscle memory for me at this point whenever a project has generated
files.

Reply:
I'll remove it from this PR since it's not elsewhere in the Whiskers generated
ports. Perhaps we could discuss it and place the recommendation there rather
than have this repository be the outlier amongst the organization.